python/pc_ble_driver_py/ble_driver.py

```diff
@@ -361,6 +361,8 @@ def from_c(cls, adv_report_evt):
         index           = 0
         while index < len(ad_list):
             ad_len  = ad_list[index]
+            if ad_len == 0:
+                break
```

Comment:
Adding break statement in this place will suppress an error condition. In my opinion we should move begining of the try block (there is appropriate log).